### PR TITLE
Methods products

### PR DESCRIPTION
**Pull request by LucasQuinotto**
🙌

### CODE REVIEW

File: API_Products/database/book_db.py [hunk 1]
```diff
@@ -1,3 +1,29 @@
+<<<<<<< HEAD
```
Comment (bublitzjr) [inline API_Products/database/book_db.py:1]: Favor resolver conflitos

**Review by jrbublitz — CHANGES_REQUESTED** [review]

File: API_Products/database/book_db.py (hunk 1 above)
Comment (LucasQuinotto) [reply API_Products/database/book_db.py]: RESOLVIDO !! 👼🏽

**Review by jrbublitz — APPROVED** [review]
ok

**Review by Thiago-Alexandre — APPROVED** [review]
Ok

File: API_Products/database/book_db.py [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+def insert_book(dict_values: dict):
+
+    if dict_values:
+        DB.book.insert_one(dict_values)
+        return "Registrado com sucesso!"
+    else:
+        raise Exception("Registros inválidos.")
+
+
+def read_all() -> list:
```
Comment (Vergiliogu) [inline API_Products/database/book_db.py:15]: read all o que? Renomeie para read_all_books.

**Review by Vergiliogu — CHANGES_REQUESTED** [review]

**Review by Vergiliogu — APPROVED** [review]

**Pull request merged**